lib/cache/Cargo.toml

```
@@ -25,6 +25,9 @@ wasmer-engine-universal = { path = "../engine-universal", version = "=2.2.1" }
 wasmer-engine-dylib = { path = "../engine-dylib", version = "=2.2.1" }
 
 [features]
+default = ["sys"]
+sys = ["wasmer/sys-default"]
+js = ["wasmer/js-default", "wasmer/js-serializable-module"]
```

I don't think we should enforce sys or js by default. We can simplify this such as:

```suggestion
default = ["wasmer/js-serializable-module"]
```

This should work without adding any feature dependency on wasmer (that is, that the wasmer crate will not automatically use `js` or `sys`, it will just use the `wasmer/js-serializable-module` feature when using js but do nothing on `sys`, which is good.

---

I was curious about this

> wasmer crate will not automatically use js or sys

Wouldn't it make existing build as breaking, that upper deps having wasmer-cache or transitive deps need to manually opt in `sys` if they're using default? Or it'll be fine since any pkg having `wasmer` already may opt in js / sys?

---

Oh, I think I was not super clear. I think the `filesystem` should be enabled by default (that way there are no breaking changes)
However, we should not enforce sys or js by default (wasmer-cache should remain agnostic).
In any case, I did applied a small suggestion and the PR looks good!